### DISCUSSION TITLE
Question about MemoryDiagnoser

### DISCUSSION THREAD

**Discussion by NWoodsman** [dotnet/BenchmarkDotNet#2440] [General]
Is MemoryDiagnoser results a sum of all test runs, or an average of a single test run? If that's the case, what is the number of runs so that I can obtain an idea of the allocation of one run?

**Comment by timcassell** [comment]
It invokes the method N times, then divides the memory measurements by N. So it's effectively the result of a single run.

**Reply by NWoodsman** [reply]
Perfect, thanks.
